aonvif/exceptions.py

```diff
@@ -4,3 +4,7 @@
 
 class ONVIFError(Exception):
     """Base type for all ONVIF errors."""
+
+
+class InvalidCapabilitiesError(Exception):
```

Comment:
Broken exception hierarchy. Probably there is no reason to use the separate exception. If wrong capabilities lead to broken client behavior then runtime exception is more appropriate.

Reply:
Replaced exception with RuntimeError